app/models/recipe.rb

```diff
@@ -2,6 +2,8 @@ class Recipe < ApplicationRecord
   validates :name, presence: true,
     uniqueness: true
   validates :description, presence: true
+  validates :img_url, presence: true
+  validates :ing_description, presence: true
```

Comment:
Of ingredient. While adding a recipe description we have diffrent boxes for typing instructions (like order pizza) and for detailed ingredients list (like box of pizza, bottle of wine)... 
It's easier to style it this way. At least that what they've told me.